### DISCUSSION TITLE
Welcome to mpv-osc-modern-f Discussions!

### DISCUSSION THREAD

**Discussion by FinnRaze** [FinnRaze/mpv-osc-modern-f#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
@FinnRaze First of all, thank you for writing this great osc, but I can't figure out what the button to the left of the info sign does?
![image](https://github.com/FinnRaze/mpv-osc-modern-f/assets/43301560/068ff058-8d7b-4aae-9422-53fc3b30359d)

**Reply by FinnRaze** [reply]
It indicates and cycles the state of "ontop"
